fog/view/server/src/db_fetcher.rs

```diff
@@ -331,6 +349,8 @@ impl<DB: RecoveryDb + Clone + Send + Sync + 'static> DbFetcherThread<DB> {
                         block_index,
                         err
                     );
+                    // We might have more work to do, we aren't sure because of the error
+                    has_more_work = true;
```

Comment:
nit: Would it make sense to rename this variable to `might_have_more_work`? Right now, the naming suggests to me that there is definitely more work.

Reply:
ok